src/mmw/js/src/draw/views.js

```diff
@@ -44,7 +44,10 @@ function actOnLayer(datum) {
 
 function validateRwdShape(result) {
     var d = new $.Deferred();
-    if (result.watershed) {
+    if (result.watershed.features[0].geometry.type === 'MultiPolygon') {
```

Comment:
Realized on the train that this guard should probably get nested inside the `if (result.watershed)` check.